apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -102,10 +106,13 @@ def render(element_html, data):
     custom_format = pl.get_string_attrib(
         element, "custom-format", CUSTOM_FORMAT_DEFAULT
     )
+    show_score = pl.get_boolean_attrib(element, "show-score", SHOW_SCORE_DEFAULT)
+    accessibility_description = pl.get_string_attrib(element, "accessibility-description", name)
```

Comment:
I don't think we want to use `name` as the default here.

Reply:
Would something like "textfield for inputting answer variable `name`" work better?

Reply:
@nwalters512 would know more. I'm not familiar with all of the accessibility options.